packages/components/Callout/src/Callout.tsx

```diff
@@ -14,14 +15,23 @@ export const Callout = compose<ICalloutType>({
   usePrepareProps: (props: ICalloutProps, useStyling: IUseComposeStyling<ICalloutType>) => {
     const { componentRef, target, ...rest } = props;
     const calloutRef = useViewCommandFocus(componentRef);
-    const targetNativeTag = findNodeHandle(target.current);
+    const [targetNativeTag, setTargetNativeTag] = React.useState<number>(null);
+
+    React.useLayoutEffect(() => {
+      if ((target !== undefined && target?.current !== null))
+      {
+        setTargetNativeTag(findNodeHandle(target.current));
+      }
+    }, [target]);
 
     const slotProps = mergeSettings<ICalloutSlotProps>(useStyling(props), {
       root: {
+        ...{
         ref: calloutRef,
-        target: targetNativeTag,
         ...rest
-      }
+      },
+      ...((targetNativeTag !== null) ? {target: targetNativeTag} : {})
+    }
```

Comment:
Can we go back to the way you had these spreads before?

`root { ref: calloutRef, target: targetNativeTag, ...rest }`

It looks a little more natural to me and uses less spreads and objects.  I think you'd only need `const [targetNativeTag, setTargetNativeTag] = React.useState<number>(**undefined**);`

Reply:
Might need to make it `useState<number | undefined>`